### DISCUSSION TITLE
Lossless audio for Sunshine?

### DISCUSSION THREAD

**Discussion by pha1n0q** [LizardByte/community#600] [Sunshine]
### Select Topic Area

Feature Request

### Body

I'd be nice to have lossless audio when streaming with Sunshine; if this is not possible, can there be a setting where we can change the audio codec and bitrate, similar in how we can change video codecs? Allowing more control over this would just be a nice thing to have!

**Comment by eShep4** [comment]
Since I often use this for music streaming, I echo this request. Did a little digging as an ageing very-rusty early-00s compsci guy, and it looks non-trivially feasible - though Moonlight would also need to be modified to accept an uncompressed PCM stream (or maybe wavpack??).

But yeah - I'd settle for the ability to change the existing OPUS encoder settings in the GUI. They're currently hardcoded and would require a recompile to modify. OPUS is inherently lossy, but if we could max out the bitrate and manually set optimization for music it'd be virtually indistinguishable for most purposes.

I wish I still had the skills to do this, but perhaps some audiophile-friendly community member could tackle it......
